lib/errors/index.js

```diff
@@ -19,8 +19,34 @@ module.exports = (
 
 			try {
 				parse(content, parser);
-			} catch ({ message }) {
-				accumulator.push({ name, message });
+			} catch (error) {
+				const { message, loc: { line, column } = {}, raisedAt } = error;
+				const issue = { name, message };
+				if (line && column) {
+					const lineOfCode = (content || '').split('\n')[line - 1];
+
+					const [ prefix, startFrom, indentation ] = column > 20
```

Comment:
Maybe worth assigning this to a meaningful variable, like `shouldTrimStart`.

Same for trimming the end.

Reply:
Just for naming the condition you think?

Reply:
I had a little bit of a hard time what this check is for, and what roles play `20` and `30`.

Reply:
Sounds good. I've added some named variables to make this more readable